systest/plugin/plugin_test.go

```diff
@@ -26,46 +26,24 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/dgraph-io/dgo/v230/protos/api"
-	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/dgraphtest"
+	"github.com/dgraph-io/dgraph/x"
 )
 
-func TestPlugins(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping test in short mode")
-	}
-
-	type testCase struct {
-		query      string
-		wantResult string
-	}
-
-	client, err := testutil.DgraphClient(testutil.SockAddr)
-	require.NoError(t, err)
-	suite := func(initialSchema string, setJSON string, cases []testCase) {
-		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
-		defer cancel()
 
-		require.NoError(t, client.Alter(ctx, &api.Operation{
-			DropAll: true,
-		}))
-		require.NoError(t, client.Alter(ctx, &api.Operation{
-			Schema: initialSchema,
-		}))
-
-		txn := client.NewTxn()
-		_, err = txn.Mutate(ctx, &api.Mutation{SetJson: []byte(setJSON)})
-		require.NoError(t, err)
-		require.NoError(t, txn.Commit(ctx))
+type testCase struct {
```

Comment:
you could use anonymous struct instead

Reply:
I revisited to incorporate this into my code, but, I need a named struct to be used at line #324 in this file-

315 func (psuite *PluginTestSuite) TestPlugins() {
316         t := psuite.T()
317
318         fmt.Printf("\nHeLLo\n\n")
319
320         if testing.Short() {
321                 t.Skip("skipping test in short mode")
322         }
323
324         pluginFn := func(initialSchema string, setJSON string, cases []testCase) {
325                 ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
326                 defer cancel()
327